Shared/MessageDialogBuilderDelegateExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace MessageDialogService
+{
+	public static partial class MessageDialogBuilderDelegateExtensions
```

Comment:
The file name and class name don't seem to have anything to do with the content of the class.